core/src/main/scala/com.snowplowanalytics.elasticsearch/loader/SnowplowTracking.scala

```diff
@@ -71,8 +74,8 @@ object SnowplowTracking {
     initialFailureTime: Long,
     message: String): Unit = {
 
-    tracker.trackUnstructEvent(SelfDescribingJson(
-      "iglu:com.snowplowanalytics.monitoring.kinesis/storage_write_failed/jsonschema/1-0-0",
+    tracker.trackSelfDescribingEvent(SelfDescribingData(
+      SchemaKey.fromUri("iglu:com.snowplowanalytics.monitoring.kinesis/storage_write_failed/jsonschema/1-0-0").get,
```

Comment:
If you know the key in advance it's better to use the constructor:
https://github.com/snowplow/iglu-scala-client/blob/f815b2692f1431af7869186b1e9ac8bcaf20f376/src/main/scala/com.snowplowanalytics.iglu/client/SchemaKey.scala#L70-L74